2022Swerve/simgui-ds.json

```diff
@@ -88,5 +88,10 @@
       "buttonCount": 0,
```

Comment:
Remove this and the other json file from git (and ideally add it to the .gitignore file)